src/jade/post/manipulate_tally.py

```
@@ -272,6 +272,50 @@ def cumulative_sum(
+            raise ValueError(
+                "Length of fwhm_frac list must match number of rows in tally."
+            )
+    else:
+        raise ValueError("fwhm_frac must be a float or a list of floats.")
+
+    Eb = tally["Energy"].values.astype(float)
+    Yb = np.zeros_like(tally["Value"])
+    sigma = np.array(fwhm_frac) * np.array(tally["Energy"]) / 2.3548
+
+    # Apply Gaussian broadening
+    for Ei, si, Yi in zip(tally["Energy"], sigma, tally["Value"]):
```

Ruff (0.13.1)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)